ivy/functional/frontends/jax/numpy/linalg.py

```diff
@@ -152,3 +151,7 @@ def cond(x, p=None):
     nan_mask = ivy.logical_and(ivy.isnan(r), ~ivy.isnan(x).any(axis=(-2, -1)))
     r = ivy.where(orig_nan_check, ivy.where(nan_mask, ivy.inf, r), r)
     return r
+
+@to_ivy_arrays_and_back
+def lstsq(a, b):
```

Comment:
I think this signature doesn't match `jax.numpy.linalg.lstsq(a, b, rcond=None, *, numpy_resid=False)` from [jax docs](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.lstsq.html). Can you please change that?